src/components/CompleteOnboardingModal.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <UnnnicModalDialog
+    :modelValue="props.showModal"
+    class="finish-onboarding-modal"
+    :primaryButtonProps="{ text: 'Ok' }"
+    hideSecondaryButton
+    showActionsDivider
+    size="sm"
+    @primary-button-click="emit('finish-onboarding')"
+  >
+    <section class="finish-onboarding-modal__content">
+      <section class="finish-onboarding-modal__content--text">
+        <h1 class="finish-onboarding-modal__content--title">
+          {{ $t('onboarding.complete_modal.title') }}
+        </h1>
+        <p class="finish-onboarding-modal__content--info">
```

Comment:
The classes of the "text", "title" and "info" elements are not correctly following the BEM standards (`block__element--modifier`), as these elements are being treated as modifiers, which can lead to confusion.